react/features/overlay/components/AbstractPageReloadOverlay.js

```diff
@@ -0,0 +1,187 @@
+import React, { Component } from 'react';
+
+import { randomInt } from '../../base/util';
+
+import { reconnectNow } from '../functions';
+import ReloadButton from './ReloadButton';
+
+declare var AJS: Object;
+declare var APP: Object;
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
```

Comment:
I can't find single import for 'jitsi-meet-logger' in the current jitsi-meet code. I prefer to keep it consistent for now.